migrations/versions/0363_cancelled_by_api_key.py

```diff
@@ -0,0 +1,45 @@
+"""
+Revision ID: 0363_cancelled_by_api_key
+Revises: 0362_broadcast_msg_event
+Create Date: 2022-02-09 14:05:27.750234
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0363_cancelled_by_api_key'
+down_revision = '0362_broadcast_msg_event'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('broadcast_message', sa.Column('created_by_api_key_id', postgresql.UUID(as_uuid=True), nullable=True))
+    op.add_column(
+        'broadcast_message', sa.Column('cancelled_by_api_key_id', postgresql.UUID(as_uuid=True), nullable=True)
+    )
+    op.drop_constraint('broadcast_message_api_key_id_fkey', 'broadcast_message', type_='foreignkey')
```

Comment:
i'm not too worried about this but this doesnt need to happen really, i think. null entries still comply with foreign key constraints, and when we drop the column later that should also remove the associated constraints with it automatically